tests/slack_sdk_async/web/test_web_client_coverage.py

```diff
@@ -568,10 +568,10 @@ async def run_method(self, method_name, method, async_method):
                 self.api_methods_to_call.remove(method()["method"])
                 await async_method()
             elif method_name == "chat_startStream":
-                self.api_methods_to_call.remove(method(channel="C123")["method"])
-                method(channel="C123", recipient_team_id="T123", recipient_user_id="U123")
-                await async_method(channel="C123")
-                await async_method(channel="C123", recipient_team_id="T123", recipient_user_id="U123")
+                self.api_methods_to_call.remove(method(channel="C123", thread_ts="123.123")["method"])
+                await async_method(channel="C123", thread_ts="123.123")
+                method(channel="C123", thread_ts="123.123", recipient_team_id="T123", recipient_user_id="U123")
+                await async_method(channel="C123", thread_ts="123.123", recipient_team_id="T123", recipient_user_id="U123")
```

Comment:
newb: There is no need to add a test that asserts that a method requires `thread_ts` since the tests will fail with a `TypeError` when a Python function's required argument is missing.